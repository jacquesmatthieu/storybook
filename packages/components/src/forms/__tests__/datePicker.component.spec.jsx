import React from 'react';
import { shallow } from 'enzyme';

import DatePicker from '../datePicker.component';

describe('DatePicker', () => {
  let props;

  beforeEach(() => {
    props = {
      input: {
        name: 'foo',
      },
      meta: {},
    };
  });

  const getWrapper = () => shallow(<DatePicker {...props} />);

  it('should render properly', () => {
    // When
    const wrapper = getWrapper();

    // Then
    expect(wrapper).toMatchSnapshot();
  });
});
