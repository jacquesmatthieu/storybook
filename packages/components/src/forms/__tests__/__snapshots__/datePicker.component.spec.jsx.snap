// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DatePicker should render properly 1`] = `
<datePickercomponent__DateInput
  input={
    Object {
      "name": "foo",
    }
  }
  meta={Object {}}
  type="date"
/>
`;
